### ISSUE TITLE
Document that Less at-import will be ignored in at-rule-*

### ISSUE THREAD

**Issue by jwilsson** [stylelint/stylelint#2703]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug which will occur in the future.

> Which rule, if any, is this issue related to?

Most (if not all) `at-rule-*` rules and probably other ones where `walkAtRules()` is used.

> What CSS is needed to reproduce this issue?

```less
@IMPORT "foo.less";
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "at-rule-name-case": ["lower"]
  }
}
```

> Which version of stylelint are you using?

`v8` branch

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`./bin/stylelint.js --config config.json file.less`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to Less.

> What did you expect to happen?

`at-rule-name-case` to flag the uppercase `@IMPORT`.

> What actually happened (e.g. what warnings or errors you are getting)?

No issue was flagged because `@import` nodes in `.less`-files aren't checked anymore. This is because they're no longer emitted as `atrule` but `import`.

I'd be happy to submit a PR fixing this across all rules.

**Comment by jeddy3** [comment]
@jwilsson Thanks again for working on this! There might not be anything to do here, other than updating the applicable rule docs, as the built-in rules only contain code for parsing and checking standard CSS. A series of `isSyntaxSyntax*` utils _whitelist_ standard CSS syntax and features before any checks are run. Support for non-standard parsing and/or features is done in plugins (see [stylelint-scss](https://github.com/kristerkari/stylelint-scss) as an example).

**Comment by jwilsson** [comment]
Alright, I can take a stab at the docs when the `postcss-less@1.0` work is done.
